v2/internal/controllers/crd_networking_loadbalancer_test.go

```diff
@@ -116,3 +107,19 @@ func Test_Networking_LoadBalancer_CRUD(t *testing.T) {
 	g.Expect(retryAfter).To(BeZero())
 	g.Expect(exists).To(BeFalse())
 }
+
+// TODO: This is still really awkward
+func getFrontendIPConfigurationARMID(tc *testcommon.KubePerTestContext, rg *resources.ResourceGroup, lbName string, lbFrontendName string) (string, error) {
```

Comment:
I kinda think that he should just return the error there rather than panicing -- since it's already asserted against w/ `g.expect` above and that feels like it'll generate a clearer error.